app/src/ui/lib/select.tsx

```diff
@@ -11,6 +11,9 @@ interface ISelectProps {
   /** The default value of the select control. */
   readonly defaultValue?: string
 
+  /** Whether or not the select control is disabled. False by default. */
+  readonly disabled?: boolean
```

Comment:
Needed to disable de email dropdown when the user chooses the "Use global git config" option.